setup.py

```diff
@@ -83,7 +83,7 @@ def get_version():
 
 def write_version_file(version, sha):
     # Exists for BC, probably completely useless.
-    with open(ROOT_DIR / "torchvision/version.py", "w") as f:
+    with open(os.path.join(ROOT_DIR, "torchvision", "version.py"), "w") as f:
```

Comment:
Thanks for the PR @bjarzemb 

We're trying to migrate away from `os.path.join`. I was hoping that `"abc/def/"` would work on Windows but maybe it doesn't. I think this should still be OK thought:

```suggestion
    with open(ROOT_DIR / "torchvision" / "version.py", "w") as f:
```

Reply:
Yes, that works too